### PR TITLE
Completed Core (korey)

### PR DESCRIPTION
**Pull request by rockettdev**

### CODE REVIEW

File: src/controllers/movie.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+
+const getAllMovies = async (req, res) => {
+
+    const movies = await prisma.movie.findMany({
+        include: {
+            screenings: true,
+          }
+    })
+    return res.status(200).json({ movies })
```
Comment (ample-samples) [inline src/controllers/movie.js:12]: .status(200) doesn't have to be included as it defaults to 200, but it's nice to see you did it anyways as it makes it clearer to see what's going on

File: src/controllers/movie.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        }
+    })
+
+    return res.status(200).send({ movie: foundMovie })
+
+    // if(foundMovie) {
+    //     res.send({movie: foundMovie});
+    //   } else {
+    //     res.send(`No movie with ID: ${id}`)
+    //   }
```
Comment (ample-samples) [inline src/controllers/movie.js:34]: Pls remove this comment as it's irrelevant to the behaviour of the code

File: src/routers/movie.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    createMovie,
+    getMovieById,
+    updateMovie
+} = require('../controllers/movie');
+
+const router = express.Router();
+
+// In index.js, we told express that the /customer route should use this router file
+// The below /register route extends that, so the end result will be a URL
+// that looks like http://localhost:4000/customer/register
```
Comment (ample-samples) [inline src/routers/movie.js:13]: These comments are also incorrect as this file deals with the movie router not customer

File: src/controllers/movie.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  }
+
+  const createMovie = async (req, res) => {
+    const { title, runtimeMins } = req.body
+
+    if (!title || !runtimeMins) {
```
Comment (ample-samples) [inline src/controllers/movie.js:40]: I like this if statement, very tidy

File: src/routers/movie.js (hunk 3 above)
Comment (rockettdev) [reply src/routers/movie.js:13]: I think this was defaulted in the code, I didn't write this haha

**Pull request closed without merging**